src/pages/_components/ExampleShowcase.tsx

```diff
@@ -4,9 +4,9 @@ import Collapse from "~/components/Collapse.jsx"
 import { positiveMod } from "~/helpers/math.js"
 import agenciesImage from "../_assets/example-agencies.png"
```

Comment:
You beat me to it!  Lets go ahead and convert these to webp as well, Lighthouse always prefers modern formats anyway

Reply:
I actually did try, but found these images had minimal gains (2–3 kB) for a visibly lower quality. So I stuck with compressing the couple that were seriously heavy.

Reply:
Oh that's interesting!  Gotta love how all over the place different compression formats are!